frontend/src/employee-frontend/components/absences/AbsenceCell.tsx

```diff
@@ -158,18 +158,21 @@ export default React.memo(function AbsenceCell({
 
   const tooltipText = useMemo(
     () =>
-      absences
-        .map(
-          ({ careType, absenceType, modifiedAt, modifiedByType }) =>
-            `${i18n.absences.careTypes[careType]}: ${
-              i18n.absences.absenceTypes[absenceType]
-            }<br/>
+      backupCare
+        ? `${i18n.absences.absenceTypes['TEMPORARY_RELOCATION']}<br/>
+            ${backupCare.date.format()}`
```

Comment:
I think this date should either be left out completely or replaced by the back up placement's update timestamp.

Reply:
OK, I'll remove it for now